bigtable-client-core-parent/bigtable-v1-protos/README.md

```diff
@@ -0,0 +1,76 @@
+# Bigtable Protos
+
+## Purpose
+This project houses the raw .proto files that are used for accessing Google Cloud Bigtable.  This project generates a jar with java artifacts derived from the proto files in this project.  These proto files can also be used to generate other clients.
+
+This project relies on artifacts from https://github.com/grpc/grpc-java. You can see more about gRPC at https://github.com/grpc/grpc-common.
+
+## Bigtable proto files
+
+There are 3 types of functionality that the protos support:
+
+* Table administration - creating, modifying and deleting Bigtable tables
+* Table data manipulation
+* Cluster administration - creating, modifying and delting Bigtable clusters, which are a collection of Tables.
+
+Here are the files directly associated with Bigtable functionality 
+* src/main/proto/google/bigtable/admin/table/v1/bigtable_table_data.proto
+* src/main/proto/google/bigtable/admin/table/v1/bigtable_table_service.proto
+* src/main/proto/google/bigtable/admin/table/v1/bigtable_table_service_messages.proto
+* src/main/proto/google/bigtable/v1/bigtable_data.proto
+* src/main/proto/google/bigtable/v1/bigtable_service.proto
+* src/main/proto/google/bigtable/v1/bigtable_service_messages.proto
+* src/main/proto/google/bigtable/admin/cluster/v1/bigtable_cluster_data.proto
+* src/main/proto/google/bigtable/admin/cluster/v1/bigtable_cluster_service.proto
+* src/main/proto/google/bigtable/admin/cluster/v1/bigtable_cluster_service_messages.proto
+
+Copied Protos
+--------------
+
+There were some proto files copied into this project from other projects.
+
+Retrieved from https://github.com/google/googleapis:
+
+* src/main/proto/google/api/annotations.proto
+* src/main/proto/google/api/http.proto
+* src/main/proto/google/longrunning/operations.proto
+* src/main/proto/google/rpc/code.proto
+* src/main/proto/google/rpc/error_details.proto
+* src/main/proto/google/rpc/status.proto
+* src/main/proto/google/type/color.proto
+* src/main/proto/google/type/date.proto
+* src/main/proto/google/type/dayofweek.proto
+* src/main/proto/google/type/timeofday.proto
+
+# Generating the java gRPC implementations
+
+Run the following:
+
+```
+mvn -P generateProtos package
+mvn -P generateServices package
+find . -name '*.java' | grep -v bigtable | xargs rm
+```
+
+# Generating the Python protos
+
+The gRPC Python generation process requires protoc 3
+
+Protoc can be installed via the instructions at:
+https://github.com/grpc/grpc-java#build-protobuf.
+Installing it isn't always trivial;
```

Comment:
The comment is out of date.  I'll fix the v1/legacy README for this commit.  I'll update the core proto doc in a separate commit.